selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -319,14 +319,15 @@ void WifiUI::refresh() {
       // SSID
       hlayout->addSpacing(50);
       QString ssid = QString::fromUtf8(network.ssid);
-      if(ssid.length() > 30){
-        ssid = ssid.left(30)+"…";
+      if(ssid.length() > 20){
+        ssid = ssid.left(20 - 3) + "…";
       }
       QLabel *ssid_label = new QLabel(ssid);
       ssid_label->setStyleSheet(R"(
         font-size: 55px;
       )");
-      hlayout->addWidget(ssid_label);
+      ssid_label->setFixedWidth(700);
```

Comment:
why add these fixed sizes everywhere? Qt handles this nicely if we give it the right constraints